devtools/jenkins/test_examples.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+envname=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1`
+sed -i -- 's/tensorflow$/tensorflow-gpu/g' scripts/install_deepchem_conda.sh
+export python_version=2.7
+bash scripts/install_deepchem_conda.sh $envname
+source activate $envname
+python setup.py install
+conda install -y jupyter
+conda install -y nbconvert
```

Comment:
Good point. Will remove those imports.

It might not be a bad idea to make jupyter part of the default install. Let's discuss on a separate issue.